docs/tutorial.rst

```diff
@@ -3,6 +3,96 @@
 ========
 Tutorial
 ========
+There are two main ways of accessing the resources of the ``stdpopsim`` package that will
+be detailed in this tutorial. The first is via the command line interface (CLI). This is
+useful if you want to do a straightforward run of the models in the ``stdpopsim``
+catalog. The other way to access the ``stdpopsim`` resources is via the python API. This
+is a bit more complicated however it allows for more advanced tasks. This tutorial will
+walk through both ways of using the ``stdpopsim`` package as well as some more advanced
+tasks you may wish to do.
+
+***************************
+Using the ``stdpopsim`` CLI
+***************************
+The CLI provides access to the catalog of models that have already been implemented by
+``stdpopsim``. The first step for using the CLI is to select the species that you are
+interested in simulating data for. In order to see which species are available run
+
+.. code-block:: console
+
+    $ stdpopsim --help
+
+This shows the species currently supported by ``stdpopsim``. This means that
+``stdpopsim`` knows various traits of these species including chromosome size and
+recombination rates. Once we've selected a species, in this case humans, we can look at
+the help again as follows.
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help
+
+For conciseness we do not show the output here but this time you should see a different
+output which shows options for performing the simulation itself and the species default
+parameters. This includes selecting the demographic model, chromosome, recombination map,
+and number of samples. The most basic simulation we can run is to draw two samples using
+the defaults (constant size population, uniform recombination map, and default
+chromosome). To save time we will also specify that the simulation use chromosome 22 with
+the ``-c`` option. 
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr22 -o foo.ts 2
+
+.. warning:: It's important to remember to either redirect the output of ``stdpopsim`` to
```

Comment:
When I saw that `2` dangling on the end, I was confused what it was referring to, but then I remembered you said two samples. I think it would be more intuitive to have all of the required positional arguments first and then add the optional flagged arguments on the end. so:
```
stdpopsim homsap 2 -c chr22 -o foo.ts
```

Reply:
But, if it will be standard to have the sample sizes at the end of the command, then you can leave it there and just drill it into people.

Reply:
I think it will be standard. Especially because it seems more meaningful if it comes after the model specification but I'm open to whatever the consensus is @gtsambos @andrewkern ?

Reply:
I'm going to leave this as is for now. Feel free to change it in a future PR if that seems better.